### DISCUSSION TITLE
Integartion with MOBIX App or iOS fetch agent  in order to receive current location

### DISCUSSION THREAD

**Discussion by ekzarov** [fetchai/uAgents#159] [Integrations]
Hello, team!

I use MOBIX app a lot. What I want is to write set of fetch agents which will analyze and work with my geolocations I used when trip with MOBIX app. In order to do that I need to track my position and need some kind of location tracer or logs of my positions when I start my trip in MOBIX app and when I finish it. 
So agent should be simple as possible - just sending current location to fetch agent with some delays and precise of geolocation.

I want to try to write something similar for android, but I think this is first agent that should be done on mobile.

Use case: I use bicycle in order to get to gym. During my route I meet semaphors and stop on them. I want fetch agent to send me sms or slack message when I should go faster or slower in order to avoid stops.

**Comment by rishankjhavar** [comment]
The bicycle use case would look so cool in action! Looking forward to this integration. 🙌

**Comment by abhifetch** [comment]
@ekzarov are you still working on this integration. If so please raise a PR or can guide us with the repo and we can carry forward the further development. And, if you are not working on it anymore we can close this discussion.
